db/schema.rb

```diff
@@ -82,13 +82,6 @@
     t.index ["user_id"], name: "index_organizers_on_user_id"
   end
 
-  create_table "participants", force: :cascade do |t|
```

Comment:
Why is this table gone?

Reply:
I am unsure about this. We had to reset our database several times but this shouldn't delete the table, should it?